data-protocols/ids/ids-core/src/main/java/org/eclipse/dataspaceconnector/ids/core/IdsCoreServiceExtension.java

```diff
@@ -94,8 +94,8 @@ public void initialize(ServiceExtensionContext serviceExtensionContext) {
             throw new EdcException(String.join(", ", settingErrors));
         }
 
-        TransformerRegistry transformerRegistry = createTransformerRegistry();
-        serviceExtensionContext.registerService(TransformerRegistry.class, transformerRegistry);
+        IdsTransformerRegistry transformerRegistry = createTransformerRegistry();
```

Comment:
`var` can be used

Reply:
as above: I also prefer `var`, but if we change it, we should change it consistently, which doesn't really fit into this PR.